### DISCUSSION TITLE
Clarify difference between default mode and WASM

### DISCUSSION THREAD

**Discussion by wong2** [vercel/satori#192] [General]
In the [Runtime and WASM](https://github.com/vercel/satori#runtime-and-wasm) section of the README, I learned that in browser, satori uses asm.js by default, but can optionally use WASM too. 
So what's the difference between these two usages? 
Does the WASM method perform better? 
What's the tradeoff to consider when choosing between these methods?

And does the WASM way work in Node.js? If so, what's the comparison between native module vs WASM?

**Comment by wong2** [comment]
Another question, why not using WASM everywhere, since browser & node.js both supports it

**Comment by shuding** [comment]
When running in the browser, the main difference is that WASM files need to be hosted and fetched, before initializing. Where asm.js can be bundled together with the lib. And then of course WASM should be faster.

When running on the Node.js server, native should be faster. However there are Node.js environments where native modules are not supported (e.g. StackBlitz's WebContainers), or other JS runtimes that support WASM (e.g. Vercel's Edge Runtime, CloudFlare's worker, or Deno).

Of course there are other difference between asm.js, native and WASM, such as security and compatibility.

Overall there are many trade-offs between each choice, and it's better to pick the one that works the best for your use case.

**Reply by wong2** [reply]
Thanks! It'll be helpful if these explanations are added to the documentation.

**Reply by shuding** [reply]
Agreed! Just created #201.
